test/sqpoll-disable-exit.c

```diff
@@ -190,7 +190,7 @@ void execute_one(void)
 int main(void)
 {
   //these pointers aren't known to asan so it'll generate a lot of errors.
-#ifndef __SANITIZE_ADDRESS__
```

Comment:
And this looks like it's changing an earlier commit, fold in with that one please. I don't want to see split changes over multiple commits that are really the same thing.